packages/ts/react-crud/test/autogrid.spec.tsx

```diff
@@ -873,25 +875,42 @@ describe('@hilla/react-crud', () => {
       });
     });
 
-    describe('refresh trigger', () => {
-      it('reloads data when increasing refreshTrigger', async () => {
+    describe('refresh grid', () => {
+      let autoGridRef: AutoGridRef;
+
+      const AutoGridRefreshTestWrapper = ({ service }: { service: ListService<any> }) => {
+        const ref = useRef<AutoGridRef>(null);
+        useEffect(() => {
+          if (ref.current) {
+            autoGridRef = ref.current;
+          }
+        }, []);
+        return (
+          <span>
+            <AutoGrid service={service} model={PersonModel} ref={ref} />
+          </span>
+        );
+      };
+
+      it('reloads data only when refresh is called', async () => {
         const service = personService();
         const listSpy = sinon.spy(service, 'list');
-        const result = render(<AutoGrid service={service} model={PersonModel} refreshTrigger={0} />);
+        const result = render(<AutoGridRefreshTestWrapper service={service} />);
         await nextFrame();
         await nextFrame();
         expect(listSpy).to.have.been.calledOnce;
 
-        // Does not refresh if refreshTrigger is not changed
-        result.rerender(<AutoGrid service={service} model={PersonModel} refreshTrigger={0} />);
+        // Does not refresh if refresh is not called
+        result.rerender(<AutoGridRefreshTestWrapper service={service} />);
         await nextFrame();
         await nextFrame();
         expect(listSpy).to.have.been.calledOnce;
```

Comment:
I think we can remove these.

Reply:
Probably, that's not necessary to re-test rendering and re-rendering effect on the call to the service here.

Reply:
Done.